docs/articles/Core/Logging.md

```diff
@@ -20,9 +20,9 @@ Features provided by the wrapper:
 
 ## Usage
 
-On level 1 components can gain access to the logging by passing themselves to the [ActivateLogging](xref:Moryx.Logging.LoggerManagement#Moryx_Logging_LoggerManagement_ActivateLogging_Moryx_Logging_ILoggingHost_)-method. Inside this method the [LoggerManagement](xref:Moryx.Logging.LoggerManagement) will create a [IModuleLogger](xref:Moryx.Logging.IModuleLogger) instance and set the Logger property of the [ILoggingHost](xref:Moryx.Logging.ILoggingHost) interface that is required by the [ActivateLogging](xref:Moryx.Logging.LoggerManagement)-method. After the call returned using the logger instance is identical to the usage described below.
+On level 1 components can gain access to the logging by passing themselves to the [ActivateLogging](xref:Moryx.Logging.LoggerManagement#Moryx_Logging_LoggerManagement_ActivateLogging_Moryx_Logging_ILoggingHost_)-method. Inside this method the [LoggerManagement](xref:Moryx.Logging.LoggerManagement) will create a [IModuleLogger](../../../src/Moryx/Logging/IModuleLogger.cs) instance and set the Logger property of the [ILoggingHost](xref:Moryx.Logging.ILoggingHost) interface that is required by the [ActivateLogging](xref:Moryx.Logging.LoggerManagement)-method. After the call returned using the logger instance is identical to the usage described below.
```

Comment:
`LoggerManagement` and `ILoggingHost` also do not exist anymore. Can this line be deleted?